service.go

```diff
@@ -49,6 +52,10 @@ func NewService(ctx context.Context) (*Service, error) {
 		return nil, err
 	}
 
+	if appConfig.Workdir == "" {
+		appConfig.Workdir = filepath.Join(xdg.DataHome, "/nostr-wallet-connect")
```

Comment:
```suggestion
		appConfig.Workdir = filepath.Join(xdg.DataHome, "/alby-nwc")
```

let's maybe use a more specific name to prevent potential future conflicts. 
@rolznz what do you think is `alby-nwc` ok?